src/lib/addons/hyperlink.js

```diff
@@ -0,0 +1,183 @@
+export function initHyperlink(CodeMirror) {
+  'use strict'
+  const shell = window.require('electron').shell
```

Comment:
Don't use electron directly. It could cause lots of problems. I thought I disabled it but apparently, I didn't do it yet. Please use `main-preload.js` and `lib/electronOnly.ts` module to expose electron and native nodejs apis.

Reply:
Oh sure, still learning the API. 

Updated the needed changes, it seems to work nicely now (also fixed macOS binding, and non-initialized getWebContentsById)

Is there a way to know the ID rather than guessing it's id=1 (since we have one window?)
Should we maybe have getCurrentWindow (or getCurrentWindowWebContents), or set id explicitly to be sure (for further changes)?

Also, should I check if we are in electron when using electorn only API?